docs/www/autotune.md

```diff
@@ -6,7 +6,7 @@ order: 1
 # Designed for performance
 
 Great news! Redpanda comes with an autotuner that detects the optimal settings for your hardware.
-To get the best performance for your hardware, set Redpanda to [production mode](./docs/production-deployment).
+To get the best performance for your hardware, set Redpanda to [production mode](https://vectorized.io/docs/production-deployment).
```

Comment:
* this docs change is unrelated to the memory utilization issue so it should be a separate commit
* but, what is it fixing? the link on the webpage seems to be ok, and having an absolute URL prevents local rendering from generating local links

Reply:
You're right. My bad, I fixed it.